### DISCUSSION TITLE
Github Copilot Student Pack

### DISCUSSION THREAD

**Discussion by Wild-in-Programming** [community/community#48016] [GitHub Education]
Dear Support Team,

I am a beneficiary of the Github Student Pack and have been using VS Code and Github Copilot for the past month, and I have found this extension to be very helpful. However, I recently opened VS Code and realized that I no longer have access to Github Copilot. When I try to sign in again, it only prompts me for a 2-month free trial and does not allow me to use the student pack.

I am seeking your assistance in resolving this issue.

Thank you.

**Comment by samus-aran** [comment]
Hi @Wild-in-Programming 👋 

We're not the support team but we do try and assist the community! 👍 

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.
We apologize for any inconvenience this may cause.
To reverify, return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."

Thank you for your understanding.

**Reply by Wild-in-Programming** [reply]
Hello there!

Thank you for letting me know about the re-verification process for my current academic status to continue receiving my GitHub Education benefits. I understand that this is an important measure to prevent fraudulent use of academic affiliation proof.

I appreciate your assistance and guidance on this matter. I will definitely return to the GitHub Education benefits page and submit my information again as requested.

Thank you for your help and for taking the time to assist the community!
